types/three/examples/jsm/lines/LineMaterial.d.ts

```diff
@@ -2,7 +2,7 @@ import { Color, MaterialParameters, ShaderMaterial, Vector2 } from '../../../src
 
 export interface LineMaterialParameters extends MaterialParameters {
     alphaToCoverage?: boolean | undefined;
-    color?: number | undefined;
+    color?: Color | undefined;
```

Comment:
Can you create this PR at https://github.com/three-types/three-ts-types? Also I think `ColorRepresentation` is the correct type here (defined [here](https://github.com/three-types/three-ts-types/blob/master/types/three/src/math/Color.d.ts#L161)).

Reply:
Moving the PR to the correct repo and closing this PR.